infrastructure/controller/pkg/db/dynamodb.go

```diff
@@ -0,0 +1,75 @@
+package db
+
+import (
+	"log"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+//NOTE: we can change this later for an interface to accept multiples
+//      providers
+
+type DB struct {
+	svc *dynamodb.DynamoDB
+}
+
+func MustNewDB() DB {
+	accessKey, keyOk := os.LookupEnv("DYNAMODB_ACCESS_KEY_ID")
```

Comment:
So this implies that we need to ship the dynamodb credentials with every open source installation, right? This must be very well documented as those values will always be in the `airy.yaml`.

A question that remains is also how we can stop people from abusing these access keys?

One solution would be to build a small public endpoint on cloud that we call from here.

Reply:
Yes this credentials must be shipped with every instance in orders to have access to the DB.

This credentials only have access to DynamoDB in read-only. Not sure that they can be abused.